server/text_generation_server/models/flash_causal_lm.py

```diff
@@ -1595,7 +1595,11 @@ def warmup(
         if max_total_tokens is None:
             if get_support_chunking():
                 model_max_length = self.tokenizer.model_max_length
-                max_position_embeddings = self.config.max_position_embeddings
+                max_position_embeddings = (
+                    self.config.max_position_embeddings
+                    if hasattr(self.config, "max_position_embeddings")
+                    else model_max_length
+                )
```

Comment:
```suggestion
                max_position_embeddings = getattr(self.config, "max_position_embeddings", model_max_length)
```

(with black formatting applied)